packages/containers/SecureMessage/steps/MessageStep.js

```diff
@@ -82,15 +83,15 @@ export const MessageStep = ({ decryptedMessage }) => {
 
   const text = String.raw`${decryptedMessage.text}`;
   const result = `${text.replace(/\n/g, '<br/>')}`;
-
+  // TODO: Should we remove dangerouslySetInnerHTML?
```

Comment:
@beshkenadze why do you think that we should remove it?

Reply:
@cu4xxi why we need it here? We don't want to put anything like HTML and show it as HTML code.